composer.json

```diff
@@ -0,0 +1,38 @@
+{
+    "name": "silverstripe/tx-translator",
+    "description": "Silverstripe TX Translator",
+    "type": "silverstripe-vendormodule",
```

Comment:
```suggestion
```
Omit the type. It's not a silverstripe vendor module.

Reply:
It is, needs to go in vendor folder (see comment below re the autoloader)

Reply:
Will composer not automatically add it to the vendor folder anyway? That's not a silverstripe-specific folder.

Reply:
Removed